<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserList</class>
 <widget class="QDialog" name="UserList">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UserList</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0" rowspan="3">
    <widget class="QTreeWidget" name="Tree_UserList">
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QPushButton" name="B_Cancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="4">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Delete Account</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="QLineEdit" name="E_NametoDelete">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="B_DeleteUser">
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" colspan="4">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Create Account</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="1">
       <widget class="QLineEdit" name="E_Password">
        <property name="echoMode">
         <enum>QLineEdit::PasswordEchoOnEdit</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="E_UserName">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Password:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="Combo_UserType">
        <item>
         <property name="text">
          <string>Visitor</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>User</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Admin</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Type:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="B_CreateUser">
        <property name="text">
         <string>Create</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="B_OK">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
